minimal.js

```diff
@@ -114,9 +114,26 @@ module.exports = function (dirname, keys, opts) {
     })
   })
 
+  const formatFeedState = (feedState) => {
+    if (feedState) {
```

Comment:
:fire: - I don't know how the `feedState` is maintained ... but I wonder if there's a chance where it could be loading and this comes through as `undefined` ... I think I want to check some of that before merging this bit

Reply:
All of the functions that call `getFeedState()` are wrapped with `waitForValidators()`, so SSB-Validate must be ready.